changelog.md

```diff
@@ -1,5 +1,79 @@
 ## 15.x
 
+### 15.0-beta2 (2025-03-15)
+
+- Feature: Snow-covered rocks are now visible (#13627)
+- Feature: Place rocks on "too steep" tiles when fixing slopes (#13462)
+- Feature: Prevent towns from upgrading individually-placed houses (#13270)
+- Feature: [Win32] 2D map scrolling (#13172)
+- Feature: NewGRF Badges (#13073)
+- Feature: Convert 32bpp-only sprites to 8bpp when needed (#11602)
+- Add: [Script] Newer Cargo Classes (#13779)
+- Add: Show hyperlink destination tooltips in text window (#13742)
+- Add: [Script] Saving/loading ScriptList (#13556)
+- Add: Press Ctrl to build diagonal canals in game mode (#13432)
+- Add: Sandbox settings to Sandbox Options window (#13268)
+- Add: Setting to allow placing houses manually in-game (#13266)
+- Add: [Script] Event for when a company's president name changes (#13208)
+- Add: Ability to toggle visibility of station signs by facility (#13207)
+- Add: [Script] ScriptEventCompanyRename (#12878)
+- Add: Ukrainian Hryvnia currency (#12877)
+- Change: Make tree placement at world generation look more organic (#13515)
+- Change: [MacOS] Put the icon in a rounded rectangle (#13446)
+- Change: [Script] GetWaypointID to return the StationID of any waypoint (#13407)
+- Change: Draw company manager face jacket after collar (#13390)
+- Change: Don't distinguish between bus and truck stops when removing them (#13384)
+- Change: [Script] Rename BridgeID to BridgeType in the script API (#13352)
+- Change: Add fonts document to help window (#13305)
+- Change: Log changes to sandbox settings (#13267)
+- Change: When player joins network company, use its name instead of number in chat (#13263)
+- Change: Disable service interval widgets for non-owned vehicles (#13260)
```

Comment:
```suggestion
- Change: Disable service interval widgets for non-owned vehicles (#13260)
+ Fix: Disable service interval widgets for non-owned vehicles (#13260)
```

Reply:
I reckon you intended to change the `Change` to `Fix`. I've also moved it to the `Fix` section.